Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocAutoCompleteReferenceValueControlObjectTest.cs

```diff
@@ -38,6 +38,7 @@
 
 namespace Remotion.ObjectBinding.Web.Development.WebTesting.IntegrationTests
 {
+  [RequiresUserInterface]
```

Comment:
Consistency: most occurances of `TestFixture` have the TestFixture attribute as the first attribute. I would like to stick to this ordering.

Reasoning: The class is first and foremost a testfixture, so that's the preamble and it feels strange to not read this as the first information. 

I created a Roslyn ticket for this: https://re-motion.atlassian.net/browse/RMTEAM-528

Disclaimer: if there is a good reason to do it the other way, we should invert the Roslyn ticket.

PS: rest of review later